### DISCUSSION TITLE
[Help Wanted] Jotai Tutorial

### DISCUSSION THREAD

**Discussion by dai-shi** [pmndrs/jotai#1638] [General] [answered]
@tomvardasca https://github.com/pmndrs/jotai/discussions/1627#discussioncomment-4357901

> Something like the Svelte tutorial?

Apart from documents, it would be nice to have an interactive tutorial something like Svelte.

Would anyone be interested in leading such a project?

**Comment by Benanna2019** [comment]
I am very new to Jotai but it could be something like the new [svelte stackblitz tutorial](https://learn.svelte.dev)

Thought it might even be easy to turn your egghead course into this type of interactive tutorial that goes in depth on Jotai 

Just spitballing ideas.

**Reply by tomvardasca** [reply]
@Benanna2019 I was thinking in something like that. We could use Next.js and Code Sandbox to achieve the same thing (like the new [React beta docs](https://beta.reactjs.org/learn/thinking-in-react)) I will put, during this week or the next one, a simple example.

**Reply by tmoutinho** [reply]
Nice idea Tomé.

**Reply by vasucp1207** [reply]
@tomvardasca, having a thing like react beta docs is a good idea, I will wait for your example as I also want to contribute to this project.

**Comment by vasucp1207** [comment]
@dai-shi, I am interested to contribute to this project, as a beginner to the **jotai**, I surprised how simple we can manage the states using jotai.

**Comment by vasucp1207** [comment]
@dai-shi, I made a demo for my idea for creating an interactive course inspired by **react** and **svelte** tutorials take a look at the live [website](https://63a21332ea76c217e7d0fdca--ubiquitous-pithivier-1b708e.netlify.app/).
Tell me your views about this, I am very excited to get started on this project.

**Reply by dai-shi** [reply]
Look like a good start. I assume what @tomvardasca is suggesting is something similar.

So, @vasucp1207 if you are willing to take the lead of this project, I will send an invitation to jotai-labs org. Please also send me DM in Discord: https://discord.gg/MrQdmzd

**Reply by vasucp1207** [reply]
@dai-shi, Yes I will happy to be to lead this project.

**Comment by dai-shi** [comment, marked as answer]
@vasucp1207 started a new project at https://github.com/jotai-labs/jotai-tutorial
We can continue using this thread for discussion and also contribute there. Thanks!
